backend/lib/spree/backend_configuration/menu_item.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'spree/preferences/configuration'
+require 'spree/backend_configuration/menu_item'
```

Comment:
These requires should not be necessary

Reply:
`require 'spree/backend_configuration/menu_item'` isn't this a circular require?

Ruby handles it fine, but we should remove just that line, the other one is a direct dependency and it's ok to keep it there 👍

Reply:
> isn't this a circular require?

Yep, thats what I thought as well

Reply:
Both are unnecessary as the file is required from a file that has both. Removed, thank you Thomas, and sorry for the misunderstanding

Reply:
No worries